lib/dbservice/groups/group.ex

```diff
@@ -4,18 +4,54 @@ defmodule Dbservice.Groups.Group do
   use Ecto.Schema
   import Ecto.Changeset
 
+  alias Dbservice.Users.Student
+  alias Dbservice.Users.User
+  alias Dbservice.Sessions.Session
+
   schema "group" do
-    field :input_schema, :map
-    field :locale, :string
-    field :locale_data, :map
+    field :name, :string
+    field :parent_id, :integer
+    field :type, Ecto.Enum, values: [:batch, :group, :cohort, :program]
```

Comment:
I've generally seen in applications that when we try to modify an enum value, it gives a lot of trouble. Can you check once the following scenario:
1. Run all the migrations
2. Put in some data in this table with some dummy values
3. Try adding another value to this enum field
4. Generate and run the migration

If these steps are running fine without any errors, then we're all good. Maybe let's create an issue for this and you can work on it separately.